shared/css/user-progress.scss

```diff
@@ -172,3 +177,51 @@ a.level_link.submitted:hover
     width: 167px;
   }
 }
+
+#landingpage.teacher-course {
+  #lesson {
+    background: none;
+    border-radius: 3px;
+    .whitetext, .smallwhitetext {
+      margin-left: 10px;
+      color: $dark_charcoal !important;
```

Comment:
HACK: All of these styles will be moved to the React components when the design is finalized..